account_account_tag_code/README.rst

```diff
@@ -0,0 +1,47 @@
+=========================
+Account Account Tag Code
+=========================
+
+This module adds a field 'code' to model 'account.account.tag'.
+This field 'code' is useful to search specific tags without having problems with the translations.
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/account-financial-tools/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us to smash it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/account-financial-tools/issues/new?body=module:%20account_account_tag_code%0Aversion:%2016.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* ForgeFlow
+
+Contributors
+~~~~~~~~~~~~
+
+* ForgeFlow Team.
```

Comment:
Usually we put the names of the developers here with the following format:
`* Marina Alapont <marina.alapont@forgeflow.com>`